lib/wavefront-metrics-formatter.js

```diff
@@ -3,60 +3,60 @@
 const delta = require('./delta');
 const Histogram = require('metrics').Histogram;
 
-function counterPoint(counter, prefix, ts, tags) {
-  if (delta.hasDeltaPrefix(counter.name)) {
+function counterPoint(counter, metricName, prefix, ts, tags) {
+  if (delta.hasDeltaPrefix(metricName)) {
     const value = counter.count;
-    const name = delta.resetAndGetName(counter, prefix, counter.name, value);
+    const name = delta.resetAndGetName(counter, prefix, metricName, value);
     return pointLine('', name, '', value, ts, tags);
   } else {
-    return pointLine(prefix, counter.name, '', counter.count, ts, tags);
+    return pointLine(prefix, metricName, '', counter.count, ts, tags);
   }
 }
 
-function gaugePoint(gauge, prefix, ts, tags) {
-  return pointLine(prefix, gauge.name, '', gauge.value, ts, tags);
+function gaugePoint(gauge, metricName, prefix, ts, tags) {
+  return pointLine(prefix, metricName, '', gauge.value, ts, tags);
 }
 
-function meterPoints(meter, prefix, ts, tags) {
+function meterPoints(meter, metricName, prefix, ts, tags) {
```

Comment:
timerPoints and meterPoints are redundant, might as well define one in terms of the other.